### PR TITLE
Portfolio changes.

### PR DESCRIPTION
**Pull request by Bickor**

### CODE REVIEW

File: portfolio/src/main/webapp/photography.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            <button onclick="randomPicture()">Get a Random Picture!</button>
+            <div id="picture-container">
+                <img src="/images/Landscape1.jpg" alt="landscape" id="landscapeImage">
+            </div>
+        </div>
+        <footer>
+            <p> © Martin Heberling</p>
+        </footer>
+  </body>
+</html>
```
Comment (tzsebe) [inline portfolio/src/main/webapp/photography.html:31]: nit: add the newline at the end of your files

File: portfolio/pom.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (tzsebe) [inline portfolio/pom.xml:37]: Looks like the TODO is done. :)

Comment (Bickor) [reply portfolio/pom.xml:37]: Done!

File: portfolio/src/main/webapp/photography.html (hunk 1 above)
Comment (Bickor) [reply portfolio/src/main/webapp/photography.html:31]: Will do.

**Pull request merged**